app/assets/javascripts/discourse/tests/helpers/qunit-helpers.js

```diff
@@ -412,7 +420,7 @@ export function fixture(selector) {
 }
 
 QUnit.assert.not = function (actual, message) {
-  deprecated("assert.not() is deprecated. Use assert.notOk() instead.", {
+  deprecated("assert.not() is deprecated. Use assert.false() instead.", {
```

Comment:
is this supposed to be here?

Reply:
yup 😄 I sneaked this small recommendation change, not that it matters since nothing is using that deprecated assertions so nobody will see it 😂

Reply:
I do the same, you know, I know 🤫